Makefile

```diff
@@ -1,14 +1,18 @@
-.PHONY: bin/hypershift-operator
+.PHONY: hypershift-operator control-plane-operator generate build test verify manifests install uninstall fmt vet docker-build docker-push install-release-images release-images-data
```

Comment:
I'm not sure I like this instead of having a separate .PHONY target above/below their corresponding target. It makes it easy to miss when adding/removing targets.

Reply:
We actually only need a PHONY declaration for one target, so I removed the top-line declaration and added PHONY only where necessary